New version: libgeotiff_jll v100.702.300+0

jlbuild opened the pull request:
Autogenerated JLL package registration

* Registering JLL package libgeotiff_jll.jl
* Repository: https://github.com/JuliaBinaryWrappers/libgeotiff_jll.jl
* Version: v100.702.300+0
* Commit: 0c42756c7a4fa366edd6eb63784b9d3621980cd8
* Revision on Yggdrasil: https://github.com/JuliaPackaging/Yggdrasil/commit/dbb7f549f8aa35901ffd1ce6adec890f49b43d28
* Created by: @visr

visr commented on jll/L/libgeotiff_jll/Compat.toml:
```diff
@@ -23,15 +23,25 @@ julia = "1.6.0-1"
+Libdl = ["0.0.0", "1"]
+Libtiff_jll = "4.7.0-4"
+PROJ_jll = "902.500.0-902"
+Zlib_jll = "1.3.0-1"
```
```suggestion
Zlib_jll = "1.2.12-1"
```

Already done in the build script: https://github.com/JuliaPackaging/Yggdrasil/pull/9589

giordano commented:
Stop, this is wrong

jlbuild committed:
New version: libgeotiff_jll v100.702.300+0

UUID: 06c338fa-64ff-565b-ac2f-249532af990e
Repo: https://github.com/JuliaBinaryWrappers/libgeotiff_jll.jl.git
Tree: f9b0609367e0dc096ebce9a5a777d7dbd6b75bf9

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
